sinks/signalfx/signalfx.go

```diff
@@ -71,13 +72,22 @@ func (sfx *SignalFxSink) Flush(ctx context.Context, interMetrics []samplers.Inte
 		}
 		dims := map[string]string{}
 		// Set the hostname as a tag, since SFx doesn't have a first-class hostname field
-		dims[sfx.hostnameTag] = sfx.hostname
+		if _, ok := sfx.excludedTags[sfx.hostnameTag]; !ok {
```

Comment:
Not your fault, but the fact that you have to do the exclusion in so many places is annoying. Sorry!

Reply:
:)